configurations/arm_instance_types/Is4gen_4xlarge.yaml

```diff
@@ -0,0 +1,4 @@
+instance_type_db: 'is4gen.4xlarge'  # The equivalent of i3.4xlarge
```

Comment:
This is incorrect!
the instance type that eqvivealnt to i3.4xlarge is **im4gn.4xlarge**.
Please edit comment, add another configuration and change all longevities below to use that configuration.

With Is4gen configuration, configure a longevity that is equivalent to **i3en**.